pkg/storage/unified/apistore/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.23.7
 
 replace (
 	github.com/grafana/grafana => ../../../..
+	github.com/grafana/grafana/apps/dashboard => ../../../../apps/dashboard
```

Comment:
@toddtreece does it make sense to do the same for all apps in the main `go.mod`?

Reply:
it seems like it'd work without it as long as the app is in the `go.work`. does it error if you remove this?

Reply:
It seems to build but there's a bunch of errors like:
```
Running go mod tidy in ./pkg/storage/unified/resource
go: downloading github.com/grafana/grafana/apps/dashboard v0.0.0-20250220154326-6e5de80ef295
go: github.com/grafana/grafana/pkg/storage/unified/resource imports
	github.com/grafana/grafana/apps/dashboard/pkg/apis/dashboard/v0alpha1: github.com/grafana/grafana/apps/dashboard@v0.0.0-20250220154326-6e5de80ef295: invalid version: missing github.com/grafana/grafana/apps/dashboard/go.mod at revision 6e5de80ef295
```

I guess that's because the code is not there in main yet but I was also put off by a bunch of similarly-looking `replace` directives here.

I can remove the new ones – let me know if you'd like me to do that.

Reply:
ah ok. it might need a follow up PR to remove once it's in main. it seems fine for this PR